source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title: Migrate to the "Live" cluster
+last_reviewed_on: 2021-07-20
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+
+This document intends to act as a guide to help you migrate your namespace from "Live-1" to "Live".
+
+### Terms used in this document
+
+There are some confusing names and terms used in this document. Here are a few:
+- "Live-1" = The name of the current Cloud Platform (Kubernetes) cluster your namespace is deployed to.
+- "Live" = The name of a new Cloud Platform (EKS) cluster that you'll need to migrate to.
+
+## Why should you migrate
+
+The Cloud Platform is a Kubernetes cluster deployed using a tool named kOps. When the Cloud Platform was conceived in 2017, Kubernetes management tools such as EKS (Elastic Kubernetes Service from AWS) and GKE (Google Kubernetes Engine) were in their infancy. An architecture decision record (ADR) was created with the understanding and acceptance of the extra management overhead and slightly slower release times. Since then, EKS has matured to a state where the benefits of running a multi-tenant cluster without the overhead of maintaining the control plane has become extremely attractive. If you wish to read more, please see this ADR.
```

Comment:
```suggestion
The migration is happening to improve the reliability and maintainability of the Cloud Platform. This is being achieved by moving from a self-managed Kubernetes cluster (using kOps), to a provider-managed cluster (using EKS). When the Cloud Platform was created in 2017, managed Kubernetes services such as EKS (Elastic Kubernetes Service from AWS) and GKE (Google Kubernetes Engine) were in their infancy. The team [accepted](https://github.com/ministryofjustice/cloud-platform/blob/main/architecture-decision-record/014-Why-we-build-our-own-kubernetes-cluster.md) the extra management overhead and slightly slower release times of self-managing, until EKS became a viable option. EKS became generally available in 2018, starting with a very basic offering and since then maturing to offer many benefits to us, outlined in [ADR#22](https://github.com/ministryofjustice/cloud-platform/blob/main/architecture-decision-record/022-EKS.md#reasoning). Cloud Platform uses GitHub for authentication via OIDC ([ADR#6](https://github.com/ministryofjustice/cloud-platform/blob/main/architecture-decision-record/006-Use-github-as-user-directory.md)), so when EKS finally added OIDC authentication in February 2021, migrating to EKS became the CP team's priority.
```